README.md

```diff
@@ -4,14 +4,14 @@
   <h3><b>Besu Italian Food Zone</b></h3>
```

Comment:
- [ ] Please Counters should be present in all the interfaces, as I can see the comments counter doesn't work directly you should Load the page to see the result please fix it.
![image](https://user-images.githubusercontent.com/74744787/226131054-407a4436-da82-4a84-8abc-a5ef54e9723d.png)